API/Utilities/ClaimsPrincipalExtensions.cs

```diff
@@ -43,6 +41,34 @@ public static bool IsUser(this ClaimsPrincipal claimsPrincipal)
     public static bool IsMatchVerifier(this ClaimsPrincipal claimsPrincipal) =>
         IsInRole(claimsPrincipal, "verifier");
 
+    /// <summary>
+    /// Denotes the principle as having either the verifier or admin role
+    /// </summary>
+    /// <param name="claimsPrincipal"></param>
+    /// <returns></returns>
+    public static bool CanVerifyMatches(this ClaimsPrincipal claimsPrincipal) =>
+        (claimsPrincipal.IsAdmin() || claimsPrincipal.IsMatchVerifier());
```

Comment:
Strictly speaking, only a user with the `verifier` scope should have this ability. In the database, we give roles in a verbose manner (i.e. "admin" does not imply "verifier"). Would like to have a better solution in the future, but for now this should be addressed.

Reply:
Gotcha, this is based off of some function I thought used to be in the claims principle extensions